system/ee/ExpressionEngine/Addons/rte/views/toolbar.php

```diff
@@ -7,13 +7,17 @@
                         <?php
                             $lc_class = str_replace(':', '', strtolower($button));
                             $id = 'tb-option-' . $lc_class;
+                            $file = PATH_THEMES . 'asset/img/rte-icons/'.$lc_class.'_icon.svg';
                         ?>
                         <span id="<?php echo $id ?>" class="cke_toolbar tb-option tb-selected">
                             <span class="cke_toolgroup">
                                 <a class="cke_button cke_button_off cke_button__<?php echo $lc_class ?> <?php if (!in_array($button, $selection)) {
                                         echo 'disabled';
                                     } ?>" title="<?php echo htmlspecialchars($label) ?>">
-                                <span class="cke_button_icon cke_button__<?php echo $lc_class ?>_icon" data-icon="<?php echo $lc_class ?>">&nbsp;</span>
+                                <span class="cke_button_icon cke_button__<?php echo $lc_class ?>_icon" data-icon="<?php echo $lc_class ?>">
+                                    <svg width="16" height="16"><?php echo file_get_contents($file); ?></svg>
```

Comment:
I think we need additional XSS sanitizing here - pinging @matthewjohns0n

Reply:
I tested this thoroughly and wasn't able to find an issue with it since the path is pre-set and the files are all coming from us.